components/Deck/TreePanel/util/TreeUtil.js

```diff
@@ -1,15 +1,17 @@
 class TreeUtil {
     //build node URL based on the context
-    static makeNodeURL(selector, page, mode) {
+    static makeNodeURL(selector, page, mode, slug) {
+        if (!slug) slug = '_';
```

Comment:
Why is this underscore needed in any case?

Reply:
I added that so that a URL would always have a slug part and to make the the URL parser in fluxible-router less prone to errors wherein it might confuse URL parts. Just a bit of defensive code. Actually there was a bug when the slug was missing which I've since fixed, but I like to keep stabilising stuff around :)

Reply:
OK. I do the same in the merged component/Util of TreeUtil and ComponentUtil.